src/proto/types.rs

```diff
@@ -647,8 +662,7 @@ impl Frame {
             out[1] = u8::try_from(self.payload.len()).expect("checked by previous branch");
             &mut out[2..6]
         };
-        // SAFETY: All ranges are exactly 4 bytes long
-        unsafe { mask_slice.try_into().unwrap_unchecked() }
+        mask_slice.try_into().unwrap()
```

Comment:
Sorry I had to push again because I forgot to pull `main`. In the process I've removed this one too, as the optimizer should easily be able to prove that all branches yield a slice of length 4

Reply:
No worries. This one should  be fairly trivial too